src/Compilers/Server/VBCSCompiler/DesktopBuildServerController.cs

```diff
@@ -104,7 +104,7 @@ protected override int RunServerCore(string pipeName, IClientConnectionHost conn
             // exit immediately with a non-zero exit code
             var mutexName = BuildServerConnection.GetServerMutexName(pipeName);
             bool holdsMutex;
-            using (var serverMutex = new Mutex(initiallyOwned: true,
+            using (var serverMutex = BuildServerConnection.OpenOrCreateMutex(initiallyOwned: true,
                                                name: mutexName,
                                                createdNew: out holdsMutex))
```

Comment:
Please align the rest of the method arguments with `initiallyOwned` as they were before.